### PR TITLE
Kevin/personal website workshop

### PR DESCRIPTION
**Pull request by Olympicene**

### CODE REVIEW

File: content/personalwebsite.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+author:
+- Kevin Cordero
+date: October 1rd, 2024
```
Comment (lowpolyneko) [inline content/personalwebsite.md:4]: think you meant `1st`

File: content/personalwebsite.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+subtitle: LUG x WICS workshop
+title: Personal Website Workshops
+---
+
+# Personal Website Workshop
+Design to Deployment
+
+## Introductions
+Hi! I’m Nandana
+- Senior majoring in CS
```
Comment (lowpolyneko) [inline content/personalwebsite.md]: the `pandoc` generation on the site is alittle strange and it ends up bunching the bullets if there isn't a space between the list and the text just above, take a look at the output here
image

File: content/personalwebsite.md (hunk 1 above)
Comment (lowpolyneko) [inline content/personalwebsite.md]: think you meant `1st`

File: content/personalwebsite.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+- Treasurer of LUG
+- Might be your 341 TA
+- Likes Linux
+- Might be good at it
+- Also built my personal website this summer.
+
+## Our Credentials
+
+Here are our personal websites:  
+Nandana : https://nandana.dev
```
Comment (lowpolyneko) [inline content/personalwebsite.md]: the urls here don't get rendered as links, this is more of an issue with the website, but as a temporary measure surround the links with the markdown `[<display text>](<url>)` as a stopgap

**Review by lowpolyneko — COMMENTED** [review]
otherwise looks good, if you wanna see how the markdown gets rendered run `make demo` in the root folder and it'll generate the website and serve it on `localhost:8080`.

when i have time after midterms i'll get around to doing this for you if you don't get around to doing it, but these are easy changes. i need to improve the website's rendering cus valid markdown like what you submitted doesn't really get translated properly and its a huge headache.

File: content/personalwebsite.md (hunk 2 above)
Comment (Olympicene) [reply content/personalwebsite.md]: Resolved this, I don't know if this behavior is intended it seems none of the list formatting Pandoc would usually handle are being handled. Fixed by manually putting in newlines.

File: content/personalwebsite.md (hunk 1 above)
Comment (Olympicene) [reply content/personalwebsite.md]: Fixed this and various other grammar mistakes

File: content/personalwebsite.md (hunk 3 above)
Comment (Olympicene) [reply content/personalwebsite.md]: Fixed urls

File: content/personalwebsite.md (hunk 2 above)
Comment (lowpolyneko) [reply content/personalwebsite.md]: pandoc markdown is kinda strange, what you have *should* render properly but it doesn't for some reason

**Comment by lowpolyneko** [comment]
lgtm now, thanks!

**Pull request merged**